File: level2/ex07.js
function solution(n) {
  // 표현 방법 수
  let count = 0;
  
  for (let i = 1; i <= n; i++) {
      // 연속된 자연수의 합
      let sum = 0;
      
      for (let j = i; j <= n; j++) {
          // 합 갱신 
          sum += j;
          // n이 표현된 경우
          if (sum === n) {
              count += 1;
              break;
          }
          // n을 표현할 수 없는 경우
          if (sum > n) {
              break;
          }        
      }
  }
  
  return count;
}

File: level2/ex07.py
def solution(n):
    # 표현 방법의 수
    count = 0
    
    for i in range(1, n+1):
        # 연속된 자연수의 합
        sum = 0
        for j in range(i, n+1):
            # 합 갱신 
            sum += j    
            # 표현한 경우
            if sum == n:
                count += 1
                break
            # 표현 못 하는 경우
            if sum > n:
                break
    return count